region/maridia/inner-yellow/Maridia Elevator Room.json

```diff
@@ -309,11 +291,12 @@
       "link": [1, 2],
       "name": "Shinespark",
       "notable": false,
+      "entranceCondition": {
```

Comment:
1<->2 can enter shinecharging and leave shinecharged.  Not many frames remaining on 1->2 however.

Reply:
I'm definitely not skilled enough to time those types of strats. Could you or @blkerby help with them?